<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>16</width>
    <height>16</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="radioButtonSimple">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Обычный</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonCompl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>105</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Инженерный</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
